android/app/src/main/res/values/styles.xml

```diff
@@ -6,4 +6,9 @@
         <item name="android:textColor">#000000</item>
     </style>
 
+    <style name="BootTheme" parent="AppTheme">
+        <!-- set the generated bootsplash.xml drawable as activity background -->
+        <item name="android:background">@drawable/bootsplash</item>
+    </style>
```

Comment:
Shouldn't we set the color of the status bar here?

```xml
<item name="colorPrimary">@color/primary</item>
<item name="colorPrimaryDark">@color/primaryDark</item>
```

Source: https://developer.android.com/guide/topics/ui/look-and-feel/themes

Reply:
@dominguesgm, did you already have the opportunity to include this?

Reply:
Sorry, I was under the impression you were going to look at it 🙁 I can take a stab at it

Reply:
Done! ✅